### PR TITLE
PRO-3379: Amend the content on the Stop Pages

### PR DESCRIPTION
**Pull request by LucaDelBuonoHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/PRO-3379

### Change description ###
Amend the content on the Stop Pages

**Does this PR introduce a breaking change?** (check one with "x")
```
[  ] Yes
[X ] No
```

### CODE REVIEW

File: app/resources/en/translation/stoppage.json [hunk 1]
```diff
@@ -3,12 +3,12 @@
...
-  "mentalCapacity": "You can&rsquo;t use the online service because not all executors are able to make their own decisions.",
-  "deathCertificate": "You need to have a <a rel=\"external noopener\" href=\"{deathCertificate}\">copy of the death certificate</a> before applying for probate.",
+  "mentalCapacity": "You can&rsquo;t use the online service as not all executors are able to make their own decisions.</p><p>Download and complete the <a rel=\"external noopener\" href=\"{applicationFormPA1P}\">paper application form (PA1P)</a>.</p><p>You can also <a rel=\"external noopener\" href=\"{guidance}\">download guidance</a> and find out where to <a rel=\"external noopener\" href=\"{registryInformation}\">send your completed form</a>.</p><p>You will need to complete a 'mental capacity form' for the executor who is unable to make their own decisions. Download it at: <a rel=\"external noopener\" href=\"https://www.gov.uk/wills-probate-inheritance/if-youre-an-executor\">https://www.gov.uk/wills-probate-inheritance/if-youre-an-executor</a>.",
```
Comment (adamb-moj) [inline app/resources/en/translation/stoppage.json]: Can we change the apostrophe's around mental capacity form to be `&lsquo;` and `&rsquo;` please.

@shepda, I'm also not too happy about the level of markup (e.g. `<p>` tags) that's going into this content.  What are your thoughts?   In most other content files we adopt a paragraph by paragraph approach, so this example would have noWillParagraph1, noWillParagraph2 and noWillParagraph3 etc (see privacy.json for an example).  Using the above format could get tricky to maintain.

Comment (LucaDelBuonoHMCTS) [reply app/resources/en/translation/stoppage.json]: Quotes replaced.

File: app/resources/en/translation/stoppage.json [hunk 2]
```diff
@@ -3,12 +3,12 @@
   "header": "Sorry, you can&rsquo;t use the online service",
   "contactProbateOffice": "<a rel=\"external noopener\" href=\"http://hmctsformfinder.justice.gov.uk/HMCTS/GetForm.do?court_forms_id=735\">Please download the probate application form (PA1), print it out and fill it in.</a>",
   "defaultReason": "Unable to proceed",
-  "noWill": "You can&rsquo;t use the online service because the person who died didn&rsquo;t leave a will.",
-  "notOriginal": "You can&rsquo;t use the online service because you don&rsquo;t have the original will of the person who died.",
-  "notExecutor": "You can&rsquo;t use the online service if you&rsquo;re not named as an executor or if you&rsquo;re applying on behalf of someone else.",
+  "noWill": "You can&rsquo;t apply for probate online because the person who died didn&rsquo;t leave a will.</p><p>Download and complete the <a rel=\"external noopener\" href=\"{applicationFormPA1A}\">paper application form (PA1A)</a>.</p><p>You can also <a rel=\"external noopener\" href=\"{guidance}\">download guidance</a> and find out where to <a rel=\"external noopener\" href=\"{registryInformation}\">send your completed form</a>.",
```
[the remark this answers is not in the document]
Comment (LucaDelBuonoHMCTS) [reply app/resources/en/translation/stoppage.json]: Again, with David we said this is not optimal but can stay for now and could create a tech debt ticket to split these messages by paragraph.

[the remark this answers is not in the document]
Comment (adamb-moj) [reply app/resources/en/translation/stoppage.json]: That's fine with me.  Thank you Luca!

**Pull request closed without merging**